odoo/tools/config.py

```diff
@@ -124,6 +124,9 @@ def __init__(self, fname=None):
         group.add_option("--upgrade-path", dest="upgrade_path",
                          help="specify an additional upgrade path.",
                          action="callback", callback=self._check_upgrade_path, nargs=1, type="string")
+        group.add_option("--pre-upgrade-scripts", dest="pre_upgrade_scripts", my_default="",
+                         help="Force specific upgrade scripts while loading base module (ignored unless -u)",
```

Comment:
NIT: `Run specific upgrade scripts before loading any module when -u is provided.`